### PR TITLE
[ART-3529] JIRA: migrate to Personal Access Tokens for authentication

### PR DESCRIPTION
**Pull request by vfreex**
Also migrates tarball-sources job to Python.

### CODE REVIEW

File: jobs/build/tarball-sources/Jenkinsfile [hunk 1]
```diff
@@ -30,30 +30,19 @@ node {
             [
                 $class: 'ParametersDefinitionProperty',
                 parameterDefinitions: [
+                    commonlib.ocpVersionParam('VERSION'),
                     string(
-                        name: 'COMPONENTS',
-                        description: '(REQUIRED) Brew component name',
-                        defaultValue: "logging-fluentd-container",
-                        trim: true,
+                        name: "ASSEMBLY",
```
Comment (thegreyd) [inline jobs/build/tarball-sources/Jenkinsfile:35]: Would this work for 3.11?

File: jobs/build/tarball-sources/Jenkinsfile [hunk 2]
```diff
@@ -70,109 +64,56 @@ node {
...
-                    withEnv(["ds=${description}"]){
-                        cmd = "artjira create -summary=\"OCP Tarball sources\" -description=\"${ds}\""
-                        jira = commonlib.shell(
-                            script: cmd,
-                            returnStdout: true
-                        )
+                cmd += [
+                    "tarball-sources",
+                    "--group", "openshift-${params.VERSION}",
+                    "--assembly", params.ASSEMBLY,
```
Comment (thegreyd) [inline jobs/build/tarball-sources/Jenkinsfile:91]: Same question for here, what happens to 3.11

Comment (vfreex) [reply jobs/build/tarball-sources/Jenkinsfile:91]: 3.11 job gets the default value `stream`.

File: jobs/build/tarball-sources/Jenkinsfile (hunk 1 above)
Comment (vfreex) [reply jobs/build/tarball-sources/Jenkinsfile:35]: Yes, just use the default value 'stream'.

**Comment by thegreyd** [comment]
@vfreex Changes look good to me. Would recommend a test run if you haven't already (putting that in the description would be nice).

**Pull request merged**